satpy/readers/netcdf_utils.py

```diff
@@ -143,13 +146,15 @@ def collect_metadata(self, name, obj):
         # Look through each subgroup
         base_name = name + "/" if name else ""
         for group_name, group_obj in obj.groups.items():
-            self.collect_metadata(base_name + group_name, group_obj)
+            full_group_name = base_name + group_name
+            self.file_content[full_group_name] = group_obj
+            self._collect_attrs(full_group_name, group_obj)
+            self.collect_metadata(full_group_name, group_obj)
         for var_name, var_obj in obj.variables.items():
             var_name = base_name + var_name
             self.file_content[var_name] = var_obj
             self.file_content[var_name + "/dtype"] = var_obj.dtype
             self.file_content[var_name + "/shape"] = var_obj.shape
-            self.file_content[var_name + "/dimensions"] = var_obj.dimensions
```

Comment:
This was added a couple of weeks ago, why remove it ?

Reply:
@sjoro Are you sure you used the latest master?  Both here and at the change around (new) line 230-231 the PR appears to revert changes that were merged into master recently, see https://github.com/pytroll/satpy/pull/1195 and https://github.com/pytroll/satpy/pull/1199.

Reply:
the `netcdf_utils`-file in this PR is not from the latest master.